simulation/test_tracker.cpp

```diff
@@ -262,6 +262,8 @@ TORRENT_TEST(announce_interval_1200)
 
 struct sim_config : sim::default_config
 {
+	explicit sim_config(bool ipv6 = true) : ipv6(ipv6) {}
```

Comment:
would you mind doing something like this instead? (or maybe even use ``boost::asio::ip::protocol``)

```
enum class ip_protocol { ipv4, ipv6 };

explicit sim_config(ip_protocol) ...
```